src/main/java/com/opentok/OpenTok.java

```diff
@@ -566,6 +568,27 @@ public StreamList listStreams(String sessionId) throws OpenTokException {
         }
     }
 
+    /**
+     * Gets a list of {@link Stream} object for the given session ID.
+     *
+     * @param sessionId The session ID.
+     * @param token The token.
+     * @param properties The SipProperties.
+     * @return The  {@link Sip} object.
+     */
+    public Sip sipDial(String sessionId, String token, SipProperties properties) throws OpenTokException {
```

Comment:
I'm not sure if we should call this method `sipDial`, I think using `dial` is cleaner and consistent with the other SDKs.